lib/string_to_ipa.rb

```diff
@@ -2,10 +2,14 @@
 require "sqlite3"
 
 module StringToIpa
-  DATABASE = SQLite3::Database.new(File.join(File.expand_path(File.dirname(__FILE__)), "..", "ipagem.db"))
-
-  DATABASE.results_as_hash = true
-  DATABASE.execute( "PRAGMA encoding = \"UTF-16\"" );
+  class << self
+    def database
```

Comment:
I would write lines 5-6 as
```
def self.database
   ...
end
```
It's primarily taste. Not sure what syntax @hilarysk prefers.